core/api/src/domain/quotes/errors.ts

```diff
@@ -0,0 +1,32 @@
+import { DomainError, ErrorLevel } from "@/domain/shared"
+
+export class QuotesError extends DomainError {}
+
+export class QuotesServiceError extends QuotesError {}
+export class QuotesNotAvailableError extends QuotesServiceError {}
+export class NoConnectionToQuotesError extends QuotesServiceError {
+  level = ErrorLevel.Critical
+}
+
+export class QuotesAlreadyAcceptedError extends QuotesServiceError {}
+export class QuotesExpiredError extends QuotesServiceError {}
+export class QuotesCouldNotParseIdError extends QuotesServiceError {}
```

Comment:
if this is not used in domain then probably the error should be difference or we are not validating correctly the quote id